tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/LongLivedTokenSetMethod.scala

```diff
@@ -0,0 +1,77 @@
+package com.linagora.tmail.james.jmap.method
+
+import com.google.inject.{AbstractModule, Scopes}
+import com.google.inject.multibindings.{Multibinder, ProvidesIntoSet}
+import com.linagora.tmail.james.jmap.json.LongLivedTokenSerializer
+import com.linagora.tmail.james.jmap.longlivedtoken.{InMemoryLongLivedTokenStore, LongLivedToken, LongLivedTokenSecret, LongLivedTokenStore}
+import com.linagora.tmail.james.jmap.method.CapabilityIdentifier.{LINAGORA_LONG_LIVED_TOKEN, LINAGORA_PGP}
+import com.linagora.tmail.james.jmap.model.{LongLivedTokenSetRequest, LongLivedTokenSetResponse, TokenCreateRequest, TokenCreateResponse}
+import eu.timepit.refined.auto._
+import org.apache.james.jmap.core.CapabilityIdentifier.{CapabilityIdentifier, JMAP_CORE}
+import org.apache.james.jmap.core.{Capability, CapabilityProperties, Invocation}
+import org.apache.james.jmap.core.Invocation.{Arguments, MethodName}
+import org.apache.james.jmap.json.ResponseSerializer
+import org.apache.james.jmap.method.{InvocationWithContext, Method, MethodRequiringAccountId}
+import org.apache.james.jmap.routes.SessionSupplier
+import org.apache.james.mailbox.MailboxSession
+import org.apache.james.metrics.api.MetricFactory
+import org.reactivestreams.Publisher
+import play.api.libs.json.{JsError, JsObject, JsSuccess, Json}
+import reactor.core.scala.publisher.SMono
+
+import javax.inject.Inject
+
+case object LongLivedTokenCapabilityProperties extends CapabilityProperties {
+  override def jsonify(): JsObject = Json.obj()
+}
+
+case object LongLivedTokenCapability extends Capability {
+  val properties: CapabilityProperties = LongLivedTokenCapabilityProperties
+  val identifier: CapabilityIdentifier = LINAGORA_LONG_LIVED_TOKEN
+}
+
+class LongLivedTokenSetMethodMethodModule extends AbstractModule {
+  override def configure(): Unit = {
+    Multibinder.newSetBinder(binder(), classOf[Method])
+      .addBinding()
+      .to(classOf[LongLivedTokenSetMethod])
+
+    bind(classOf[InMemoryLongLivedTokenStore]).in(Scopes.SINGLETON)
+    bind(classOf[LongLivedTokenStore]).to(classOf[InMemoryLongLivedTokenStore])
```

Comment:
Have the binding to the actual implementation done in another guice module please.